### PR TITLE
Array shift

### PR DESCRIPTION
**Pull request by William-Moreno**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/arrayShift/array-shift.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  newArray.push(val);
+
+  for(let i = secondLoopStart ; i < arr.length ; i++){
+    newArray.push(arr[i]);
+  }
+
+  return newArray;
+};
+
```
Comment (adrianhuebner) [inline challenges/arrayShift/array-shift.js:22]: Looks really great, there is a way to do it without .push() that would be great knowledge to have
